cypress/support/fragments/invoices/settingsInvoices.js

```diff
@@ -49,7 +49,7 @@ export default {
     cy.do(MultiColumnListCell({ content: batchGroup.name }).perform(
       element => {
         const rowNumber = element.parentElement.parentElement.getAttribute('data-row-index');
-        const createdByAdmin = `${DateTools.getFormattedDateWithSlashes({ date: new Date() })} by ADMINISTRATOR, DIKU`;
+        const createdByAdmin = `${DateTools.getFormattedDateWithSlashes({ date: new Date() })} by folio-aqa`;
```

Comment:
Please use more universal approach. Take the Last name[, First name] from the profileDropdown